pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -686,6 +686,9 @@ func allocatableResources(memory, cpu string) v1.ResourceList {
 
 // DaemonSets should not place onto nodes with insufficient free resource
 func TestInsufficientCapacityNodeDaemonDoesNotLaunchPod(t *testing.T) {
+	if utilfeature.DefaultFeatureGate.Enabled(features.ScheduleDaemonSetPods) {
```

Comment:
as `utilfeature.DefaultFeatureGate.Enabled(features.ScheduleDaemonSetPods)` is used multiple times, maybe storing the value to a local variable in `init()` is better?

Reply:
Sure, I can look into it. I am much more concerned about the PR as such, does it make sense to skip them or remove them? I personally feel that these tests should be in till this feature becomes GA.